config/defaults/oauth2provider.json

```diff
@@ -226,7 +226,7 @@
     "authorisedIdmDelegationClients": []
   },
   "clientDynamicRegistrationConfig": {
-    "dynamicClientRegistrationSoftwareStatementRequired": true,
+    "dynamicClientRegistrationSoftwareStatementRequired": false,
```

Comment:
Why false??? That sounds contrary to FAPI requirements?

Reply:
IAM can't parse the JWT to validate the ISS attribute because the SSA issued by Open Banking don't comply with the standard and IAM raise and error parsing the SSA JWT, now is IG who validates the SSA issued by OB.
@see the issue related: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/96